### DISCUSSION TITLE
transform中添加修改值的功能 Function of adding and modifying values in transformer

### DISCUSSION THREAD

**Discussion by Iatbzh** [grafana/grafana#73258] [Dashboards, Panels & Visualizations]
Add the function of modifying values in dashboard>panel>transform, for example, in the data I collected, the label is "instance: 192.168.0.2:9100", but I don't want to display it as 192.168.0.2:9100. I want to remove ": 9100" and hope to remove it by selecting the label to modify and then removing ": 9100"

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82564. Feel free to continue the discussion around this there. Thank you!
